doc/tutorials/intermediate/structure_datastore.md

```diff
@@ -0,0 +1,300 @@
+# Structure Applications with a DataStore
```

Comment:
I've migrated the `DataStore` into the ´intermediate` tutorial.

I've tried to structure it such that it can run with pyodide in this notebook. But also such that its a *real* example of an app that is structured into multiple files. We don't really have that today. And its one of the things that make Panel hard to use because users have to think about and invent their own structure.

The app lacks some finish. Visually as shown below. Use readable names, use the fill width, align better etc. But **most important** when you change values in the widgets a lot of warnings and errors are raised in the terminal.

I hope that @philippjfr would finish the app up?

Reply:
And I should consider whether you want to use `.rx` for this example. It might be more understandable and robust with @pn.bind/ @pn.depends.